ui/appui-react/src/appui-react/widget-panels/Content.tsx

```diff
@@ -37,21 +37,36 @@ function WidgetFallback() {
 export function WidgetContent() {
```

Comment:
Is `WidgetContent` the best place to fire this event? I would have expected it to be fired by the component that contains the reparenting logic.

From a DOM perspective, it should be fired by the stable `<div>` that actually gets moved around.

Reply:
Moved event dispatching to a place where the DOM node is being reparented.